SourceX/DiabloUI/art.cpp

```diff
@@ -31,38 +31,34 @@ void LoadArt(const char *pszFile, Art *art, int frames, SDL_Color *pPalette)
 		format = 0;
 		break;
 	}
-	SDL_Surface *art_surface = SDL_CreateRGBSurfaceWithFormat(SDL_SWSURFACE, width, height, bpp, format);
+	SDLSurfaceUniquePtr art_surface { SDL_CreateRGBSurfaceWithFormat(SDL_SWSURFACE, width, height, bpp, format) };

```

Comment:
Wouldn't it be possible to do this with `unique_ptr<SDL_Surface>` syntax?

Reply:
That would be `std::unique_ptr<SDL_Surface, SDLSurfaceDeleter>` which is verbose and error-prone

Reply:
Using just `std::unique_ptr<SDL_Surface>` is not possible - specializing / defining things in `std` namespace is not allowed